internal/cli/down.go

```diff
@@ -75,7 +75,31 @@ func (c *downCmd) down(ctx context.Context, downDeps DownDeps, args []string) er
 		return err
 	}
 
-	entities, err := engine.ParseYAMLFromManifests(tlr.Manifests...)
+	if err := deleteK8sEntities(ctx, tlr.Manifests, downDeps, c.deleteNamespaces); err != nil {
```

Comment:
i feel bad about continuing to add more and more complicated logic to `tilt down`.

possible idea: once enable/disable services is out, change the implementation of `tilt down` to create all resources initially disabled, and wait until the reconcilers reach a steady state of disabling them. maybe this is weird though